test/work-items/work-backends.ts

```diff
@@ -281,13 +281,12 @@ describe('Work Backends', function () {
         for (const updateState of Object.values(WorkItemStatus).filter(k => k !== WorkItemStatus.CANCELED)) {
           describe(`And an attempt is made to update the work-item to state "${updateState}"`, async function () {
             before(async function () {
-              this.job.status =
-                this.workItem.status = updateState;
-              this.response = await updateWorkItem(this.backend, this.workItem);
+              this.workItem.status = updateState;
+              await updateWorkItem(this.backend, this.workItem);
             });
-            it('fails the update', function () {
-              expect(this.response.status).to.equal(409);
-              expect(this.response.text).to.equal(`Job was already ${terminalState}.`);
+            it('fails the update', async function () {
```

Comment:
Not sure if "fails the update" accurately describes what's being tested here anymore?

Reply:
Never mind, I think I get the gist!

Reply:
I just re-worded it to "fails to update the work item"